### DISCUSSION TITLE
Lotw certificate is uploading, but not registering

### DISCUSSION THREAD

**Discussion by w8rjd** [magicbug/Cloudlog#2225] [General]
I am having an issue. My LoTW certificate is uploading and I can see it in the uploads file, but Cloudlog still says "You need to upload some LoTW p12 certificates to use this area." under Available Certificates. Any help would be appreciated.

73 de W8RJD

**Comment by magicbug** [comment]
I'm going to hazard a guess its todo with this https://github.com/magicbug/Cloudlog/issues/2068
